modules/Hoot/ui/modals/importDataset.js

```diff
@@ -369,11 +369,17 @@ export default class ImportDataset {
 
                 return Hoot.api.statusInterval( this.jobId );
             } )
-            .then( resp => {
+            .then( async resp => {
                 let message;
                 if (resp.data && resp.data.status === 'cancelled') {
                     message = 'Import job cancelled';
 
+                    // Delete the newly created folder
+                    if ( newFolderName ) {
+                        await Hoot.api.deleteFolder( folderId );
+                        await Hoot.folders.removeFolder( folderId );
+                    }
+
```

Comment:
We talked about how this would be better done within the services.  If we moved folder creation to within a job command we could have that occur after successful clipping and if it was canceled the new output folder would never be created so we would not have to clean it up at all.  However, the services don't have a mechanism for passing input between job commands, so the folder id created by the AddFolder command would not be available to the UpdateParent command that moves output maps underneath said folder.
So we'll keep this way for now as it should run immediately after canceling.

Reply:
Note: moved this to https://github.com/ngageoint/hootenanny/issues/3412